src/client-adapters.js

```diff
@@ -128,7 +128,8 @@ class ClientAdapters {
     let localMaAdapter = new AlpheiosTuftsAdapter({
       category: 'morphology',
       adapterName: 'tufts',
-      method: options.method
+      method: options.method,
+      clientId: options.clientId
```

Comment:
As I could see - clientId is only passed to constructors and saved to it - but not used.
So it would be some future changes to handle with it?